app/assets/stylesheets/mumuki_laboratory/application/modules/_terms.scss

```diff
@@ -0,0 +1,44 @@
+summary.terms-summary {
```

Comment:
Perhaps we should avoid the `summary` selector to keep it more semantic

Reply:
Actually I removed altogether as they were for an old version :smile: